core/src/main/resources/templates/browseAcls.html

```diff
@@ -638,7 +638,7 @@ <h4 class="card-title">Topic : {{ topicSelectedParam }}</h4>
 									</ul>
 								</td>
 								<td align="right">
-									<a href="requestAcls?topicname={{topicSelectedParam}}">
+									<a href="requestAcls?topicname={{topicSelectedParam}}&?envId={{topicOverview[0].envId}}">
```

Comment:
```suggestion
									<a href="requestAcls?topicname={{topicSelectedParam}}&envId={{topicOverview[0].envId}}">
```

'?' is not needed I think